components/policies/development/integration/bootstrap-namespace/bootstrap-namespace.yaml

```diff
@@ -19,6 +19,12 @@ spec:
           selector:
             matchLabels:
               konflux-ci.dev/type: tenant
+          operations:
+          - CREATE
+          - UPDATE
```

Comment:
I suspect the declaration of `operations` is related to the chainsaw test failure. These should be the default values according to the [docs](https://kyverno.io/docs/policy-types/cluster-policy/match-exclude/#match-statements) so I'm not sure why. It was something I encountered with #7108 but didn't investigate any further.

Does the policy still work if you omit these?